pow/src/main/java/tech/pegasys/teku/pow/exception/Eth1RequestExceptionsContainer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.pow.exception;
+
+import com.google.common.base.Throwables;
+import java.net.SocketTimeoutException;
+import java.util.ArrayList;
+import java.util.Optional;
+
+public class Eth1RequestExceptionsContainer extends RuntimeException {
```

Comment:
Convention is that exception names always end with `Exception` so I think this should just be `Eth1RequestException` or even just `Eth1Exception`.